graspologic/utils/utils.py

```diff
@@ -701,3 +705,51 @@ def plug_in_variance_estimator(x):
         return covariances
 
     return plug_in_variance_estimator
+
+
+def remap_labels(y_true, y_pred, return_map=False):
+    """
+    Attempts to remap a categorical labeling (such as one predicted by a clustering
```

Comment:
I would reword this - attempts is too nebulous and can imply a variety of things you may not intend.  If you do intend attempts, I'd describe the optimal circumstance as well as an example or examples of the negative cases too where we didn't succeed in automatically relabeling.

Reply:
Addressed. I expanded a little bit at the top and added a `Notes` section. If you want more than what is in the notes currently, I think it'd be easier at that point to just make an actual tutorial. I thought that was overkill for this function but I do already have a notebook linked at the top.